File: java/DaySeven/TaskSeven.java
public class TaskSeven{
	public static void main(String[] args){
	
int check = 1;
int counter = 1;
int multiples = 1;
int sum = 0;

for (check = 1; check <= 10; check++){
	if (check % 4 == 0){

		while(counter <= 5){
		counter++;		
		multiples = multiples * check;
		sum = sum + multiples;
		
		
	
		}
	System.out.print(sum + " ");
}
		
		counter = 1;
		multiples = 1;
		sum = 0;
	}
}
}

File: python/DaySeven/Task_seven.py
check = 1
counter = 1



for check in range(1,11):
	if (check % 4 == 0):
		
		multiples = 1
		sum = 0
		for counter in range(5):		
			multiples = multiples * check
			sum += multiples
		print(sum, end= " ")
		

		